crates/matrix-sdk-crypto/src/store/crypto_store_wrapper.rs

```diff
@@ -126,6 +126,24 @@ impl CryptoStoreWrapper {
 
         self.store.save_changes(changes).await?;
 
+        // If we updated our own public identity, log it for debugging purposes
+        for updated_identity in
```

Comment:
Can you wrap the loop in a `if tracing::level_enabled!(tracing::level::Debug)`?

Reply:
063c62d6047db500f40479a1f17aa554cdef16dd